### PR TITLE
[RFR] Add "more-to-see" indicators when there is data hidden

### PR DESCRIPTION
**Pull request by floo51**
Opinion / Early review wanted

more-to-see-indicators

### CODE REVIEW

File: src/isAfter.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export const isAfter = (date, dateBounds) => {
```
Comment (jpetitcolas) [inline src/isAfter.js]: Use `date-fns.isAfter` instead of reinventing the wheel.

File: src/indicator.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    indicators
+        .enter()
+        .append('text')
+        .classed('indicator', true)
+        .attr('opacity', 0.5)
+        .attr('x', d => (d === 'before' ? labelWidth : '100%'))
+        .attr('dx', d => (d === 'before' ? 0 : -25))
+        .attr('y', lineHeight / 2)
+        .attr('dy', '0.25em')
+        .text(d => (d === 'before' ? '◀' : '▶'));
```
Comment (jpetitcolas) [inline src/indicator.js]: Add in config:

```
indicators: {
    previousText: "◀",
    nextText: "◀ reverse side"
}
```

And allow it to be false to disable the feature

**Comment by floo51** [comment]
Review applied

File: docs/configuration.md [hunk 3]
```diff
@@ -239,6 +239,24 @@ const chart = eventDrops({
 });
 ```
 
+## indicators
```
Comment (jpetitcolas) [inline docs/configuration.md]: Use singular, as in `label`

File: src/withinRange.js [hunk 4]
```diff
@@ -1,7 +1,8 @@
+import isWithinRange from 'date-fns/is_within_range';
+
 export const withinRange = (date, dateBounds) => {
     const startingDate = Math.min(...dateBounds);
     const endingDate = Math.max(...dateBounds);
 
-    // @TODO: remove the `new Date()` constructor in the next major version: we need to force it at configuration level.
```
Comment (jpetitcolas) [inline src/withinRange.js]: Let the `@TODO`

**Comment by jpetitcolas** [comment]
Thanks! :+1:

**Pull request merged**